src/jitterentropy-noise.c

```diff
@@ -46,7 +46,7 @@ static uint64_t jent_loop_shuffle(struct rand_data *ec,
 	(void)ec;
 	(void)bits;
 
-	return (1U<<min);
+	return (1UL<<min);
```

Comment:
Could you change `1UL` to `UINT64_C(1)` here? I think that is the correct value. That should resolve to the correct type across platforms.